src/main/kotlin/no/nav/medlemskap/common/CorrelationId.kt

```diff
@@ -13,5 +14,9 @@ class CorrelationId(private val id: String) {
     override fun toString(): String = id
 }
 
-internal fun getCorrelationId(): CorrelationId =
-    CorrelationId(MDC.get(MDC_CALL_ID))
+internal fun getCorrelationId(): CorrelationId {
+    if (MDC.get(MDC_CALL_ID) == null) {
+        return CorrelationId(callIdGenerator.get())
```

Comment:
Er dette nødvendig når getCorrelationId bare blir brukt av exceptionhandler ?

Reply:
Åja, ser nå at det var denne du snakker om på slack, der denne trengs for at cucumber testene skal bli fornøyde